cypress/e2e/exchange.cy.js

```diff
@@ -0,0 +1,95 @@
+const URL = 'http://127.0.0.1:8080'
+const API_URL = 'https://api.exchangerate.host/'
+const VALID_DATE = '2014-08-15'
+const BASE = 'ARS'
+const CONVERT_TO = 'USD'
+const TODAYS_YEAR = 2023
+
+context('Exchange rate app', () => {
+  before(() => {
+    cy.visit(URL)
+  })
+
+  describe('Test the application', () => {
```

Comment:
I think you should decide which block you will use, `describe` and `context` are the same. I prefer describe block over context.